tests/resources/MLTdata-preproduction.ecsv

```diff
@@ -1,209 +1,214 @@
 # %ECSV 1.0
 # ---
 # datatype:
-# - {name: mirror_panel_id, datatype: string, description: mirror panel ID}
-# - {name: mirror_panel_radius, unit: m, datatype: float32, description: mirror panel radius (2f)}
-# - {name: psf_opt, unit: 'mm', datatype: float32, description: Average spot size of MLT mirror panels from 2f measurement (d80), best value (not at nominal distance)}
-# - {name: psf, unit: 'mm', datatype: float32, description: Average spot size of MLT mirror panels from 2f measurement (d80) at nominal distance of 32.14 m}
+# - {name: mirror_id, datatype: string, description: mirror panel ID}
```

Comment:
Why did you go from `mirror_panel_id` to `mirror_id`? I think we wanted to keep always the reference to panel when dealing with mirror panels and not full telescope mirrors.

Reply:
This seems to be not consistent also in the schema files. `mirror_list.schema.yml` uses for instance `mirror_id`, wheras `MST_mirror_2f_measurements.schema.yml` uses `mirror_panel_id`. I tried to use mirror_id throughout but we can change this @GernotMaier ?

Reply:
I think the entry in `mirror_list.schema.yml` should change. I assume @GernotMaier will do it as part of the effort he mentioned today. Here we should follow the `MST_mirror_2f_measurements.schema.yml` example and use `mirror_panel_id`.

Reply:
Feel free to change when you work on it! Better to correct information as soon as you know that it is wrong/incomplete.